electron/src/electron/modals.ts

```diff
@@ -0,0 +1,143 @@
+import { WebPlugin } from "@capacitor/core";
+
+declare var require: any
+const {dialog , getCurrentWindow} = require('electron').remote
+
+import {
+  ModalsPlugin,
+  AlertOptions,
+  PromptOptions,
+  PromptResult,
+  ConfirmOptions,
+  ConfirmResult,
+  ActionSheetOptions,
+  ActionSheetResult
+} from "@capacitor/core";
+
+export class ModalsPluginElectron extends WebPlugin implements ModalsPlugin {
+  resolve: any;
+  constructor() {
+    super({
+      name: 'Modals',
+      platforms: ['electron']
+    });
+  }
+  async alert(options: AlertOptions): Promise<void> {
+    //window.alert(options.message); we can't set title and button title
```

Comment:
When I read this, it doesn't mean anything to me, so better remove comments like this